.github/ISSUE_TEMPLATE/Bug-Report.yml

```
@@ -0,0 +1,46 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please check the [devstack troubleshooting guide](https://edx.readthedocs.io/projects/open-edx-devstack/en/latest/troubleshoot_general_tips.html) and the [existing list of blocking bugs](https://github.com/openedx/devstack/labels/blocker) before filing a new issue.
+  - type: textarea
+    id: bug-report
+    attributes:
+      label: Describe the bug that you are seeing.
+    validations:
+      required: true
+  - type: input
+    id: container
+    attributes:
+      label: Did this happen on your machine or in the container?
```

What do you think of the following?
```suggestion
      label: Did this happen on the host (your machine or the remote instance), or in the container?
```